CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Master
 
+* **BREAKING** Removes the implicit Dangerfile work. This is a WIP. The feature will be back in some form. - orta
```

Comment:
yeah, it looks like this PR will have to provide all the docs and the replacement - so this will get a real changelog entry